Jellyfin.Server/Migrations/MigrationOptions.cs

```diff
@@ -10,14 +13,12 @@ public class MigrationOptions
         /// </summary>
         public MigrationOptions()
         {
-            Applied = System.Array.Empty<string>();
+            Applied = new List<Guid>();
         }
 
-#pragma warning disable CA1819 // Properties should not return arrays
         /// <summary>
-        /// Gets or sets the list of applied migration routine names.
+        /// Gets the list of applied migration routine names.
         /// </summary>
-        public string[] Applied { get; set; }
-#pragma warning restore CA1819 // Properties should not return arrays
+        public List<Guid> Applied { get; }
```

Comment:
Will this break the nightlies that already contain the names here?

Reply:
I tested locally and the existing string names are just ignored (not loaded)